src/extensions/nuget/Microsoft.DotNet.UpgradeAssistant.Extensions.NuGet/NuGetTransitiveDependencyIdentifier.cs

```diff
@@ -25,10 +28,18 @@ namespace Microsoft.DotNet.UpgradeAssistant.Extensions.NuGet
 {
     public class NuGetTransitiveDependencyIdentifier : ITransitiveDependencyIdentifier
     {
+        private const string UniquePathPart1 = "dotnet-ua";
+        private const string UniquePathPart2 = "dotnet-ua";
+        private const string UniquePathFilename = "dotnet-ua";
+
```

Comment:
Assuming its copy-paste error that all 3 strings have the same value?

Reply:
Actually, no - they were used in 3 different ways but all given the same value. I didn't want to assume they _should_ all be the same value so did this. If you're saying they should be, happy to consolidate :)

Reply:
These are all part of creating a scoped project file for restore. Doesn't really matter what the names are, just that the full path is unique for each invocation. Probably worth putting it in a separate method `CreateUniquePath()` or something so it's clearer what it's doing.